app/minutes/controllers.py

```diff
@@ -146,12 +146,11 @@ def create_minute(user, user_data):
     # Get the members role.
     membership = committee.members.filter_by(member= user).first()
 
-    if (user.id != committee.head and user.is_admin == False and
-        (membership is None or membership.role != Roles.MinuteTaker)):
+    if membership is None and not user.is_admin:
         emit('create_minute', Response.PermError)
         return
     
-    if user.id != committee.head and not user_data["private"]:
+    if not user_data["private"] and (user.id != committee.head or not user.is_admin):
```

Comment:
Also, is there a reason sometimes we are checking for the existence of "private" while other times we just grab it from the array directly?

Reply:
It was like this initially but I do agree that it makes more sense to use a get with a default value. Maybe the initial intention was to protect against malformed requests?

Reply:
Yeah, that was probably it. I think it would be a nice and easy way to improve readability; having the same logic checks made in the same way.